root/static/ng_templates/ontology_term_comment_transfer.html

```diff
@@ -14,7 +14,8 @@
 
   <div class="curs-workflow-fig-or-table" ng-show="showFigureField">
     <label for="figure-field">Figure or table:</label>
-    <input id="figure-field" type="text" ng-model="figOrTable"/>
+    <input id="figure-field" type="text" ng-model="figOrTable" placeholder="Figure 1, Table 1"/>
```

Comment:
Is it worth tweaking that to:
```
eg. Figure 1, Table 1, ...
```
?

I don't have a strong opinion.

Reply:
I think that's a better choice. At first I thought it would make the figure field inconsistent with the placeholders in the other fields, as shown below:

![image](https://user-images.githubusercontent.com/37659591/93187125-5afccb80-f737-11ea-9414-58346738e425.png)

But the other placeholders are more like labels for what goes in the field, rather than examples, so this isn't really a sensible objection.

I'm usually not keen on being explicit about the fact placeholders are examples, but I think in this case it's possible that curators could mistake the placeholder text for text that has actually been entered &ndash; particularly when using Edit or Copy and Edit &ndash; so clearly marking the text as an example will solve that. I'll make this change. Thanks for the suggestion.